oic.r.amacl.raml

```diff
@@ -40,15 +40,7 @@ traits:
                     "rt": ["oic.r.temperature"],
                     "if": ["oic.if.s"]
```

Comment:
i suppose ned to confirm about where amacl and sacl should they be refering to ace2.resource or oic-link?